StripComments/StripComment.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+
+namespace StripComments
+{
+    class StripComment
+    {
+        public static string StripComments(string text, string[] commentSymbols)
+        {
+            var lines = text.Split('\n');
+
+            for (int i = 0; i < commentSymbols.Length; i++)
+            {
+                lines = lines.Select(x => x.Substring(0, (x.Contains(commentSymbols[i]) ? x.IndexOf(commentSymbols[i]) : x.Length)).TrimEnd()).ToArray();
```

Comment:
Please pay attension to using `.ToArray()` etc because you are using them in almost everywhere. In most places they are not needed. Like here.